lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java

```diff
@@ -112,28 +112,28 @@ private static QuizQuestion makeQuestion5() {
             AnswerChoice.B, "Storage",
             AnswerChoice.C, "RAM",
             AnswerChoice.D, "CPU"),
-        AnswerChoice.UNANSWERED); // Replace `UNANSWERED` with the correct answer.
+        AnswerChoice.A); // Replace `UNANSWERED` with the correct answer.
   }
 
   private static QuizQuestion makeQuestion6() {
     return new QuizQuestion(
         6,
         "What is the decimal number 34 in binary?",
-        "0b00000000"); // Replace the correct digits in the answer.
+        "100010"); // Replace the correct digits in the answer.
```

Comment:
The answer is formatted incorrectly. Use the format I've provided for you in the answer.